src/default-xpring-client.ts

```diff
@@ -159,11 +170,70 @@ class DefaultXpringClient implements XpringClientDecorator {
    * @returns A promise which resolves to a string representing the hash of the submitted transaction.
    */
   public async send(
-    _amount: BigInteger | number | string,
-    _destination: string,
-    _sender: Wallet,
+    amount: BigInteger | number | string,
+    destination: string,
+    sender: Wallet,
   ): Promise<string> {
-    throw new Error(XpringClientErrorMessages.unimplemented)
+    if (!Utils.isValidXAddress(destination)) {
+      return Promise.reject(
+        new Error(XpringClientErrorMessages.xAddressRequired),
+      )
+    }
+
+    const classicAddress = Utils.decodeXAddress(sender.getAddress())
+    if (!classicAddress) {
+      return Promise.reject(
+        new Error(XpringClientErrorMessages.xAddressRequired),
+      )
+    }
+
+    const normalizedAmount = DefaultXpringClient.toBigInt(amount)
+
+    const fee = await this.getMinimumFee()
+    const accountData = await this.getAccountData(classicAddress.address)
+    const lastValidatedLedgerSequence = await this.getLastValidatedLedgerSequence()
+
+    const xrpDropsAmount = new XRPDropsAmount()
+    xrpDropsAmount.setDrops(normalizedAmount.toJSNumber())
+
+    const currencyAmount = new CurrencyAmount()
+    currencyAmount.setXrpAmount(xrpDropsAmount)
+
+    const destinationAccount = new AccountAddress()
+    destinationAccount.setAddress(destination)
+
+    const payment = new Payment()
+    payment.setDestination(destinationAccount)
+    payment.setAmount(currencyAmount)
+
+    const account = new AccountAddress()
+    account.setAddress(sender.getAddress())
+
+    const transaction = new Transaction()
+    transaction.setAccount(account)
+    transaction.setFee(fee)
+    transaction.setSequence(accountData.getSequence())
+    transaction.setPayment(payment)
+    transaction.setLastLedgerSequence(
+      lastValidatedLedgerSequence + ledgerSequenceMargin,
```

Comment:
So do we need to use `lastValidatedLedgerSequence.getIndex()` here instead? This is similar to that balance thing we were talking about yesterday.

Reply:
`getLastValidatedLedgerSequence()` returns a `number` already, so this is a `number` + `number`.

Reply:
My bad on this one, the old legacy protobuffs return an object.